### PR TITLE
Add recipe for acrv-datasets

### PR DESCRIPTION
**Pull request by btalb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This package is part of the [Best of ACRV](https://roboticvision.org/best-of-acrv). I am happy to take care of maintenance duties for this feedstock.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Hi @conda-forge/help-python - this is @btalb's first package. He's a colleague of mine and doesn't have rights to tag teams yet, so I'm doing it.

/cc @wolfv - FYI, @btalb is planning to put up quite a few robotics related packages up in the next few days. They are part of the legacy of the Australian Centre for Robotic Vision.

File: recipes/acrv-datasets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Wrapper for easy dataset access (used in Best of ACRV codebases)'
+  description: |
+    The ACRV Datasets package is a light wrapper for generically managing
+    datasets. The package supports any dataset, as long as it has a public URL.
+    We emphasise that we do not own the datasets accessed through this package,
+    we simply provide easy access and integration for projects like the Best of
+    ACRV codebases (see https://roboticvision.org/best-of-acrv).
+  doc_url: https://github.com/best-of-acrv/acrv_datasets
```
Comment (dopplershift) [inline recipes/acrv-datasets/meta.yaml:53]: Pretty sure both of these should be `acrv-datasets`.

File: recipes/acrv-datasets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  imports:
+    - acrv_datasets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/acrv-datasets/
+  summary: Datasets manager for the Best of ACRV
```
Comment (dopplershift) [inline recipes/acrv-datasets/meta.yaml:39]: All three of these are duplicated (with different vaules) below.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Looks good overall, just a couple minor things.

**Comment by btalb** [comment]
Thanks @dopplershift for catching those mistakes. Should be resolved now.

**Pull request merged**